EKrut-Server/src/ekrut/server/db/TicketDAO.java

```diff
@@ -17,34 +17,28 @@
 public class TicketDAO {
 	
 	private DBController con;
-	private TicketDAO ticketDAO;
+	
 	
 	//Constructor - provides connection con to a new TicketDAO so we can use DBcontroller
 	public TicketDAO(DBController con) {
 		this.con = con;
-		ticketDAO=new TicketDAO(con);
 	}
 	
 	
-	/** NOT FINISHED - NEED TO CONTINUE 
+	/**  
 	 * Create new ticket line in DB and set its details
-	 * @param ticket - the ticket that we want to add to DB
+	 * @param Ticket - the ticket that we want to add to DB

```

Comment:
ticket was correct. The first thing in the param tag is the name of the parameter which in this case is `ticket`, not `Ticket`